client/src/helpers/APICalls/getProfilePhoto.ts

```diff
@@ -0,0 +1,21 @@
+import { FetchOptions } from '../../interface/FetchOptions';
+
+interface ImageSourceData {
+  success: { imageURI: string };
+  error: { message: string };
+}
+
+const getProfilePhotoURI = async (): Promise<ImageSourceData> => {
+  const fetchOptions: FetchOptions = {
+    method: 'GET',
+    headers: { 'Content-Type': 'application/json' },
```

Comment:
A content-type header isn't needed here (see [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type) for details)